source/macos/AdaptiveCards/AdaptiveCardsTests/Renderers/InputDateRendererTests.swift

```diff
@@ -55,3 +90,10 @@ class InputDateRendererTest: XCTestCase {
     }
 }
 
+private extension NSButton {
+    func performClick() {
+        if let target = target, let action = action {
```

Comment:
```suggestion
    func performClick() {
        guard !isHidden else { return }
        if let target = target, let action = action {
```
Maybe add this here. I see `performClick()` should not work if the button is hidden.

Reply:
Oh thanks Akaash